bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -254,7 +254,11 @@ <h3 id="os-operators-headline" class="tween">
               <div>
                 {% if l10n_has_tag('mwc_2015') %}
```

Comment:
Okay, will remove the tag. Actually there are still a couple of MWC 2015 traces on this page  :stuck_out_tongue: